netlify.toml

```diff
@@ -4,3 +4,8 @@
   base = "code/"
   publish = "build/"
   command = "npm run build"
+
+[[redirects]]
```

Comment:
oh, nice! so here you redirect in case of error instead of with useNavigate? Curious, what is the difference between the two ways of doing it?